net/api/test.cc

```diff
@@ -135,13 +136,12 @@ class UseLocalHTTPTestServer {
       HTTPServerConnection connection(socket.Accept());
       const auto& message = connection.Message();
       const string method = message.Method();
-      const string url = message.URL();
+      const URL url = URL(message.Path());
       if (method == "GET") {
-        if (url == "/get") {
-          connection.SendHTTPResponse("DIMA");
-        } else if (url == "/drip?numbytes=7") {
+        if (url.path == "/drip") {
+          const size_t numbytes = atoi(url["numbytes"].c_str());
```

Comment:
Very good point! Let me do this once I am down to the ```express```-style server implemenation.

(It would have ```query``` parameters and the ```URL``` ones separately, ex. ```/user/:uid/followers?time_filter=1day``` should set both some ```.query["time_filter"]``` and ```.params["uid"]```.)